client/post-editor/post-editor.jsx

```diff
@@ -379,6 +379,11 @@ const PostEditor = React.createClass( {
 			this.setState( this.getInitialState(), function() {
 				this.refs.editor.setEditorContent( '' );
 			} );
+		} else if ( this.state.isNew && this.state.hasContent && ! this.state.isDirty ) {
+			// is a copy
```

Comment:
Ideally we would only use comments to clarify “why” we’re doing something, not what. I realize you followed the convention from above, but still, how do you think we can fix that?